app/src/sharedTest/java/org/oppia/app/completedstorylist/CompletedStoryListActivityTest.kt

```diff
@@ -152,6 +159,33 @@ class CompletedStoryListActivityTest {
     }
   }
 
+  @Test
+  fun testCompletedStoryList_clickOnItem0_intentIsCorrect() {
```

Comment:
Do we need to add a test case where we can check clicking on the item after configuration change also opens the activity?
This may prevent future errors.

Reply:
thanks. Added another test.

Reply:
Thanks